plugins/net_plugin/net_plugin.cpp

```diff
@@ -1516,6 +1516,7 @@ namespace eosio {
       // notices of previously unknown blocks or txns,
       //
       fc_dlog(logger, "got a notice_message from ${p}", ("p",c->peer_name()));
+      bool can_fwd = false;
```

Comment:
can we get a comment here or below, indicating we are not currently setting it?

Reply:
It's going to take a little more time to figure out the safe way to determine whether or not to forward notifications.